tests/test_meta/test_grouped_transformer.py

```diff
@@ -0,0 +1,203 @@
+
+import pytest
+import numpy as np
+import pandas as pd
+from sklearn import clone
+from sklearn.linear_model import LinearRegression
+from sklearn.preprocessing import StandardScaler, MinMaxScaler
+from sklearn.utils import check_X_y
+
+from sklego.common import flatten
+from sklego.meta import GroupedTransformer
+from tests.conftest import transformer_checks, nonmeta_checks, general_checks
+
+
+@pytest.fixture
+def tests_to_skip():
+    return [
+        # Nonsense checks because we always need at least two columns (group and value)
+        "check_fit2d_1feature",
+        "check_fit2d_predict1d",
+    ]
+
+
+@pytest.mark.parametrize(
+    "test_fn", flatten([transformer_checks, nonmeta_checks, general_checks])
+)
+def test_estimator_checks(test_fn, tests_to_skip):
+    if test_fn.__name__ not in tests_to_skip:
```

Comment:
What do you think of this approach for skipping tests from the list? I think it looks cleaner than copying the other list and filtering the ones you don't want, do you agree? Otherwise I'll change it.

One downside is that we get some 'false positives', the tests that are skipped are counted as passed test.

Reply:
I actually *really* like this, it feels much cleaner to me. @MBrouns?

Reply:
It does feel good, but i'd rather have it inside the `parametrize` than i n the test function itself. something like:

@pytest.mark.parametrize(
    "test_fn", tests(
         include=[transformer_checks, nonmeta_checks, general_checks], 
         exclude=["check_fit2d_1feature", "check_fit2d_predict1d"]
    )
)
```